### DISCUSSION TITLE
求助，错误代码变来变去

### DISCUSSION THREAD

**Discussion by rootoy** [spencerwooo/onedrive-vercel-index#799] [Q&A]
如图，第一步时这样：
<img width="934" alt="屏幕截图 2022-11-26 110437" src="https://user-images.githubusercontent.com/57469144/204070185-065e6fb4-9c87-4b58-b383-48501a1bcb49.png">

第三步时变成这样：

<img width="932" alt="屏幕截图 2022-11-26 110211" src="https://user-images.githubusercontent.com/57469144/204070191-5b605c3f-9c9b-44da-b1c8-04f651e9552e.png">

**Comment by spencerwooo** [comment]
感谢赞助，首先第二张图里的报错信息说明你的自定义的两个 `client_id` 和 `obfuscatedClientSecret` 有问题，因此如果非必要不用修改 `client_id` 和 `obfuscatedClientSecret`，然后再尝试将 Upstash 里面的两个 token 删掉重新部署一下。

因为本项目将逐步向 Next.js 13 过渡，因此我可能近期没办法及时回复问题等，请见谅。

**Comment by rootoy** [comment]
然后再尝试将 Upstash 里面的两个 token 删掉重新部署一下。
请问一下怎样将将 Upstash 里面的两个 token 删掉？

从 Windows 版邮件发送

发件人: Spencer Woo
发送时间: 2022年11月28日 10:13
收件人: spencerwooo/onedrive-vercel-index
抄送: rootoy; Author
主题: Re: [spencerwooo/onedrive-vercel-index] 求助，错误代码变来变去 (Discussion #799)

感谢赞助，首先第二张图里的报错信息说明你的自定义的两个 client_id 和 obfuscatedClientSecret 有问题，因此如果非必要不用修改 client_id 和 obfuscatedClientSecret，然后再尝试将 Upstash 里面的两个 token 删掉重新部署一下。
因为本项目将逐步向 Next.js 13 过渡，因此我可能近期没办法及时回复问题等，请见谅。

**Comment by spencerwooo** [comment]
Upstash 管理面板里的 Data Browser 可以直接删掉某个 key 的存储内容。

**Comment by rootoy** [comment]
里面好像没有值
从 Windows 版邮件发送

发件人: Spencer Woo
发送时间: 2022年11月28日 12:27
收件人: spencerwooo/onedrive-vercel-index
抄送: rootoy; Author
主题: Re: [spencerwooo/onedrive-vercel-index] 求助，错误代码变来变去 (Discussion #799)

Upstash 管理面板里的 Data Browser 可以直接删掉某个 key 的存储内容。

**Comment by spencerwooo** [comment]
那么不必修改 client_id 和 obfuscatedClientSecret，使用项目提供的两个重新部署即可。

**Comment by rootoy** [comment]
重新部署后仍然报错
从 Windows 版邮件发送

发件人: Spencer Woo
发送时间: 2022年11月28日 16:19
收件人: spencerwooo/onedrive-vercel-index
抄送: rootoy; Author
主题: Re: [spencerwooo/onedrive-vercel-index] 求助，错误代码变来变去 (Discussion #799)

那么不必修改 client_id 和 obfuscatedClientSecret，使用项目提供的两个重新部署即可。

**Comment by spencerwooo** [comment]
看到你的 GitHub 仓库并没有更新，所以你需要重新 fork 一遍仓库，然后重新修改除了 client_id 和 obfuscatedClientSecret 外的配置，再进行部署。

**Comment by rootoy** [comment]
解决了，在线加密工具中使用的密钥这一个选项应该填onedrive-vercel-index



---原始邮件---
发件人: "Spencer ***@***.***&gt;
发送时间: 2022年11月28日(周一) 下午4:19
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [spencerwooo/onedrive-vercel-index] 求助，错误代码变来变去 (Discussion #799)




 
那么不必修改 client_id 和 obfuscatedClientSecret，使用项目提供的两个重新部署即可。

**Comment by rootoy** [comment]
README文件无法显示404






---原始邮件---
发件人: "Spencer ***@***.***&gt;
发送时间: 2022年11月28日(周一) 下午4:19
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [spencerwooo/onedrive-vercel-index] 求助，错误代码变来变去 (Discussion #799)




 
那么不必修改 client_id 和 obfuscatedClientSecret，使用项目提供的两个重新部署即可。
